contracts/upgradeable_contracts/omnibridge_nft/BasicNFTOmnibridge.sol

```diff
@@ -86,8 +86,10 @@ abstract contract BasicNFTOmnibridge is
     ) external onlyMediator {
         address bridgedToken = bridgedTokenAddress(_token);
         if (bridgedToken == address(0)) {
-            if (bytes(_name).length == 0) {
-                require(bytes(_symbol).length > 0);
+            if (bytes(_name).length == 0 && bytes(_symbol).length == 0) {
+                _name = "Bridged NFT";
```

Comment:
May we leave such tokens without name and symbol (have them empty)?

Reply:
Do you know how Blockscout will handle such tokens with empty name and symbol?

Reply:
But the EIP1155 standard does not require to have symbols and names, so the block explorers must behave correspondingly.

Reply:
Ok, let's fallback to empty symbol and name if they both are missing in the origin contract.